cmd/server/config.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/kubernetes-incubator/cri-o/server"
+	"github.com/opencontainers/runc/libcontainer/selinux"
+	"github.com/urfave/cli"
+)
+
+const (
+	ocidRoot   = "/var/lib/ocid"
+	conmonPath = "/usr/libexec/ocid/conmon"
+	pausePath  = "/usr/libexec/ocid/pause"
+)
+
+var commentedConfigTemplate = template.Must(template.New("config").Parse(`
+# Commented version of ocid configuration file. Generated automatically.
+
+# The "server" table contains all of the server options. There are no
+# bare options. "server" contains global options that don't really
+# relate to any particular sub-table.
+[server]
+
+# root is a path to the "root directory" where all information not
+# explicitly handled by other options will be stored.
+root = "{{ .Root }}"
+
+# sandbox_dir is the directory where ocid will store all of its sandbox
+# state and other information.
+sandbox_dir = "{{ .SandboxDir }}"
+
+# container_dir is the directory where ocid will store all of its
+# container state and other information.
+container_dir = "{{ .ContainerDir }}"
+
+# The "server.api" table contains settings for the kubelet/gRPC
+# interface (which is also used by ocic).
+[server.api]
+
+# listen is the path to the AF_LOCAL socket on which cri-o will listen.
+# This may support proto://addr formats later, but currently this is just
+# a path.
+listen = "{{ .Listen }}"
+
+# The "server.runtime" table contains settings pertaining to the OCI
+# runtime used and options for how to set up and manage the OCI runtime.
+[server.runtime]
+
+# runtime is a path to the OCI runtime which ocid will be using. Currently
+# the only known working choice is runC, simply because the OCI has not
+# yet merged a CLI API (so we assume runC's API here).
+runtime = "{{ .Runtime }}"
+
+# conmon is the path to conmon binary, used for managing the runtime.
+conmon = "{{ .Conmon }}"
+
+# selinux determines whether or not SELinux handling is enabled
+# daemon-wide. The default is determined by whether or not SELinux
+# is enabled on the system.
+selinux = {{ .SELinux }}
+
+# The "server.image" table contains settings pertaining to the
+# management of OCI images. Currently this is a stub, as cri-o does not
+# yet have full support for handling OCI images.
+[server.image]
+
+# pause is the path to the statically linked pause container binary, used
+# as the entrypoint for infra containers.
+pause = "{{ .Pause }}"
+`))
+
+func DefaultConfig() *server.Config {
+	return &server.Config{
+		Root:         ocidRoot,
+		Runtime:      "/usr/bin/runc",
```

Comment:
As with selinux I was mirroring the old functionality. I'd be fine with using `/usr/sbin/runc` (and I would actually prefer that because that's the path that openSUSE uses :wink:).